src/router.ts

```diff
@@ -39,16 +39,6 @@ const routes: RouteRecordRaw[] = [
 const router = createRouter({
   history: createWebHistory(),
   routes,
-  scrollBehavior(to) {
```

Comment:
No more scrolling?

Reply:
fixed, and now it scrolls only on path change